openshift_cli_installer/cli.py

```diff
@@ -211,7 +211,7 @@
     is_flag=True,
     show_default=True,
 )
-def main(**kwargs):
+def installer(**kwargs):
```

Comment:
this will break other repos which currently call main

Reply:
renamed